### ISSUE TITLE
MineOS crash upon page access

### ISSUE THREAD

**Issue by clowrym** [hexparrot/mineos-node#249]
With the latest updates I am now having an issue with access to the MineOS web page. Anytime a command is issued via the web portal, mineOS quits & requires a refresh of the page with the command not working. I have attached the mineOS log.
I can log in fine, Click the drop down on each server and view the main pages, but as soon as I try and make any changes, mines exits.

I have also removed my Docker image & container files & reinstalled to make sure there were no conflicts from previous installations. This is happening on 2 different unraid systems

Docker unraid log:

`USER_NAME not provided; defaulting to "mc"
Created user: mc (uid: 1000)
Generating Self-Signed SSL...
Generating a 1024 bit RSA private key
.........++++++

+
writing new private key to '.tmpkey.pem'
-----
writing RSA key
2017-06-15 19:40:58,309 CRIT Supervisor running as root (no user in config file)
2017-06-15 19:40:58,309 WARN Included extra file "/etc/supervisor/conf.d/mineos.conf" during parsing
2017-06-15 19:40:58,319 INFO RPC interface 'supervisor' initialized
2017-06-15 19:40:58,319 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2017-06-15 19:40:58,320 INFO supervisord started with pid 1
2017-06-15 19:40:59,322 INFO spawned: 'mineos' with pid 31
2017-06-15 19:41:00,708 INFO success: mineos entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-06-15 19:41:45,352 INFO exited: mineos (exit status 1; not expected)
2017-06-15 19:41:46,355 INFO spawned: 'mineos' with pid 109
2017-06-15 19:41:47,563 INFO success: mineos entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
 `
[mineos.txt](https://github.com/hexparrot/mineos-node/files/1079518/mineos.txt)

**Comment by hexparrot** [comment]
I looked through your logs and I identified an instance of `node-uuid` (deprecated module) that still was present but should've been replaced with `uuid`.  For the sake of expediency, I re-added the module and the hub.docker image is rebuilding--in the next few minutes it should be  done and it should be back up and running.

Evidently, even after changing the module dep in `package.json`, I failed to manually delete the directory in `node_modules` giving me a false sense that everything was GTG.

Should be working momentarily.
